packages/clay-color-picker/src/index.tsx

```diff
@@ -345,12 +345,12 @@ const ClayColorPicker: React.FunctionComponent<IProps> = ({
 								insetBefore
 								onChange={(event) => {
 									onValueChange(event.target.value);
-									setCustomEditorActive(false);
-									setInternalActive(false);
+									// setCustomEditorActive(false);
```

Comment:
@sergiojimcos looks like you forgot to "clean up" here

Reply:
Oh sorry, I haven't seen that 😅